docs/src/pages/guides/composition/composition.md

```diff
@@ -10,9 +10,10 @@ To solve this problem we tag some of our components when needed
 with a `muiName` static property.
 
 However, users like to wrap components in order to enhance them.
-That can conflict with our `muiName` solution.
+That can conflict with our `muiName` solution. If you wrap a component verify if
+that component has this static property set.
 If you encounter this issue, you need to:
-1. Forward the properties.
+1. Hoist these properties.
```

Comment:
I disagree that this is more common terminology. (So does Google if you search for it in the context of React). It also risks confusion with hoisting in the JS sense.

Reply:
What's hoisting in the js sense?

Maybe "copy over" like stated in the react docs? https://reactjs.org/docs/higher-order-components.html#static-methods-must-be-copied-over

Reply:
[hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting>


The React docs also talk about "passing props down", "and passing props through", so how about:

"Pass the child properties through the wrapper component"

or similar?

Reply:
We are not talking about component props but static properties. This is what the linked section is talking about.